src/firefoxos/FileProxy.js

```diff
@@ -67,10 +69,12 @@ QUIRKS:
 
     exports.requestFileSystem = function(successCallback, errorCallback, args) {
         var type = args[0];
-        var size = args[1];
+        //var size = args[1];
```

Comment:
Please either remove commented lines, or add an additional message why this commented out but not removed